test/lapack.jl

```diff
@@ -687,7 +687,11 @@ end
     @testset for elty in (Float32, Float64, ComplexF32, ComplexF64)
         local n = 10
         a = randn(elty, n, n)
-        A = a'*a
```

Comment:
This guarantees a positive semidefinite matrix, but in practice it is positive-definite with probability ≈ 1, no?

If the problem is just an eigenvalue that is too close to zero, why not just `A = a'*a + I`?

Reply:
Yes, I think this approach should also work